documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,141 @@
+   2. The registration period(s) must be a clearly defined time range with a start and end time and date.
+      1. The first registration period must have a start time and date at least 48 hours after the competition is announced.
+         1. The competition's Delegate(s) and/or organizers may register for the competition prior to the first registration period.
+      2. If there is more than one registration period, the organization team must announce each additional registration period at least 48 hours before it starts.
+      3. Different registration periods may have different registration requirements (e.g. restricting which events a competitor can register for).
+   3. Different locations for multi-location competitions may have different registration requirements (e.g. competitor limit).
+7. Events
+   1. Each event must have a planned number of rounds.
+      1. Each round of each event must have a planned format and time limit.
+      2. A round of an event must have planned advancement conditions if there is a subsequent round of the event.
+      3. A round may have a cutoff. A round with a cutoff is a combined round, with the formats as per [WCA Regulation 9b)](wca{regulations/#9b}).
+   2. Only results from events that appear on the competition website in the Events tab and Schedule will be recognized.
+      1. Other events may only be held unofficially.
+8. Schedule
+   1. The schedule must be published on the WCA website using the WCA schedule tool.
+9. Championships
+   1. A championship type is assigned to competitions that are National, Continental, or World Championships.
+      1. A competition must be located in the region where the championship type is selected.
+      2. A competition may have multiple championship types.
+      3. A championship type for a region must only be assigned to one competition in a year.
+      4. An additional selection process is conducted by the WCA Board for organizing Continental and World Championships.
+   3. Naming
+      1. Continental and World Championships names must begin with "WCA".
+      2. The word "Championship" must be singular in the competition name.
+      3. Competitions which are not National, Continental, or World championships should avoid names that are similar to championship names.
+   4. Date(s)
+      1. National Championships should not be held on the same date(s) as Continental or World championships.
+      2. Continental Championships must not be held on the same date(s) as World Championships.
+   5. Championships should have all the officially recognized WCA events.
+   6. Championships must recognize and award champion titles and their respective prizes to the winners in the region.
+   7. Organization teams of National Championships should involve all the Delegates of the respective countries, at the discretion of the Senior Delegate.
+10. Optional requirements
```

Comment:
```suggestion
10. Optional requirements (Approval from the WCA Competition Announcement Team must be given)
```

or something of the sort.

Reply:
I think this is already explicit from 10.2019.1:
2.1.4 Approving the adoption of optional Regulations by organization teams to facilitate
the administration of competitions.

Reply:
If you think a change is due, perhaps it can be 2.1.4 Approving (or rejecting)... in the Motion.

Reply:
How about we do both (change the motion and make it explicit here)?

Reply:
I'd be good with that!

Reply:
I think this would look better as a sub-point instead of as it is suggested now, no?

Reply:
Sure, that sounds reasonable.

Reply:
I'm pushing a new version now with this change included!